src/PostgREST/Config/Database.hs

```diff
@@ -154,7 +154,7 @@ queryRoleSettings prepared =
       select
         kv.rolname,
         i.value as iso_lvl,
-        array_agg(row(kv.key, kv.value)) filter (where key <> 'default_transation_isolation') as role_settings
+        coalesce(array_agg(row(kv.key, kv.value)) filter (where key <> 'default_transaction_isolation'), '{}') as role_settings
```

Comment:
I tried putting the `'default_transaction_isolation'` into a variable and interpolate with `qc` - this fails to compile.

This has happened a number of times now, we need a new quasiquoting lib.